test/quantization/test_qat.py

```diff
@@ -2086,9 +2085,14 @@ def test_quantize_api_nvfp4(self, use_per_tensor_scale: bool):
         """
         from torchao.prototype.mx_formats import NVFP4InferenceConfig
 
+        if use_per_tensor_scale:
+            target_prepare_sqnr = 36
+        else:
+            target_prepare_sqnr = float("inf")
+
         self._test_quantize_api_against_ptq(
             NVFP4InferenceConfig(use_dynamic_per_tensor_scale=use_per_tensor_scale),
-            target_prepare_sqnr=12,
+            target_prepare_sqnr=target_prepare_sqnr,
```

Comment:
can you explain this I would expect that per-tensor scaling would have less error and thus a higher sqnr? but also I dont know what prepare sqnr means?

Reply:
This actually refers to the SQNR between (1) QAT fake quantized model and (2) PTQ actual quantized model, so a lower SQNR here just means these two don't match exactly.

I'm not sure why per tensor is not inf here. I suspect there's more logic in this case so there's larger chance for divergence. I did also try calling `NVFP4Tensor` directly for QAT just for a sanity check, but I wasn't able to get inf even with that.